src/dlio_postprocessor.py

```diff
@@ -190,9 +193,9 @@ def process_loading_and_processing_times(self):
         logging.info(f"Computing overall stats")
 
         # Save the overall stats
-        self.overall_stats['samples/s'] = self.get_stats(all_samples_per_s)
-        self.overall_stats['sample_latency'] = self.get_stats(all_sample_latencies)
-
+        self.overall_stats['samples/s'] = self.get_stats(self.summary['metric']['train_throughput_samples_per_second'])
```

Comment:
This feels very verbose. I prefer the apis to be simpler. Just get the data u want and then use that value next.